src/main/java/it/eng/idsa/dataapp/service/impl/ProxyServiceImpl.java

```diff
@@ -266,12 +266,10 @@ public ResponseEntity<String> proxyMultipartForm(ProxyRequest proxyRequest, Http
 					eccProperties.getBrokerQuerryContext(), null, null);
 		}
 
-		logger.info("Forwarding form POST request to {}", thirdPartyApi.toString());
```

Comment:
Any specific reason for deleting this info?

Reply:
Same log line is 2 lines below :)